great_expectations/core/expectation_suite.py

```diff
@@ -1133,7 +1107,7 @@ def render(self) -> None:
 class ExpectationSuiteSchema(Schema):
     expectation_suite_name = fields.Str()
     ge_cloud_id = fields.UUID(required=False, allow_none=True)
-    expectations = fields.List(fields.Nested(ExpectationConfigurationSchema))
+    expectations = fields.List(fields.Nested("ExpectationConfigurationSchema"))
```

Comment:
Don't love it but seems to work: https://stackoverflow.com/a/35002969

Reply:
Does it not work to just import his in a type checking block?